src/client/src/orangehrmRecruitmentPlugin/components/ActionHistoryLayout.vue

```diff
@@ -116,92 +118,102 @@
+import {formatDate, parseDate} from '@/core/util/helper/datefns';
 import useLocale from '@/core/util/composable/useLocale';
 import useDateFormat from '@/core/util/composable/useDateFormat';
-import {formatDate, parseDate} from '@/core/util/helper/datefns';
+
+const actionModel = {
+  candidate: null,
+  vacancy: null,
+  hiringManager: null,
+  status: null,
+  performedAction: null,
+  performedDate: null,
+  performedBy: null,
+  notes: null,
+};
 
 export default {
-  name: 'ShortlistHistoryScreen',
-  setup() {
+  name: 'ViewActionHistory',
+
+  props: {
+    candidateId: {
+      type: Number,
+      required: true,
+    },
+    historyId: {
+      type: Number,
+      required: true,
+    },
+  },
+
+  setup(props) {
     const {locale} = useLocale();
     const {jsDateFormat} = useDateFormat();
 
     const http = new APIService(
-      'https://c81c3149-4936-41d9-ab3d-e25f1bff2934.mock.pstmn.io',
-      'recruitment/candidateHistory',
+      window.appGlobal.baseUrl,
+      `/api/v2/recruitment/candidates/${props.candidateId}/history`,
     );
     return {
       http,
       locale,
       jsDateFormat,
     };
   },
+
   data() {
     return {
       isLoading: false,
-      history: {
-        candidate: '',
-        vacancy: '',
-        manager: '',
-        performedAction: '',
-        performedBy: '',
-        performedDate: '',
-        note: '',
-        status: null,
-      },
+      action: {...actionModel},
     };
   },
   computed: {
     getPerformedDate() {
       return formatDate(
-        parseDate(this.history.performedDate),
+        parseDate(this.action.performedDate),
         this.jsDateFormat,
         {locale: this.locale},
       );
     },
   },
   beforeMount() {
     this.isLoading = true;
-    this.http.getAll().then(({data: {data}}) => {
-      const {
-        candidate,
-        performedAction,
-        vacancy,
-        status,
-        manager,
-        ...rest
-      } = data;
-      const {firstName, lastName, middleName} = candidate;
-      const fullName = `${manager.firstName} ${manager.middleName} ${manager.lastName}`;
-      this.history = {
-        candidate: `${firstName} ${middleName} ${lastName}`,
-        vacancy: vacancy.title,
-        manager:
-          (manager?.terminationId ? this.$t('general.past_employee') : '') +
-          fullName,
-        performedAction: performedAction.label,
-        cid: candidate.id,
-        status: status.label,
-        ...rest,
-      };
-      this.isLoading = false;
-    });
+    this.http
+      .get(this.historyId)
+      .then(response => {
+        const {data} = response.data;
+        (this.action.candidate = `${data.candidate.firstName} ${data.candidate.middleName} ${data.candidate.lastName}`),
+          (this.action.vacancy = data.vacancy.name),
+          (this.action.hiringManager = `${data.vacancy.hiringManager.firstName}
+          ${data.vacancy.hiringManager.middleName} ${data.vacancy.hiringManager.lastName}`),
```

Comment:
any idea to use `const {$tEmpName} = useEmployeeNameTranslate();`. assuming hiringManager also an employee

Reply:
Will add in a follow up PR